nodebuilder/share/cmd/share.go

```diff
@@ -72,7 +71,7 @@ var sharesAvailableCmd = &cobra.Command{
 }
 
 var getSharesByNamespaceCmd = &cobra.Command{
-	Use:   "get-by-namespace [extended header, namespace]",
+	Use:   "get-by-namespace [block height, namespace]",
```

Comment:
Please also make it work with block hash as well, so its either height or hash

Reply:
gotcha

Reply:
@Wondertan as the issue described just using height for now, let's merge this as is. Eventually we can add support for hash or height IMO

Reply:
It's trivial to add and I don't see why we wouldn't while we are on it

Reply:
@bao1029p, could you give us an estimate and tell if you will be able to finish this until Monday(included)?

Reply:
i just done with this, will push it now